components/buttons/OkDynamicStylesButton.vue

```diff
@@ -21,23 +17,29 @@
     import { BehaviorSubject } from "~/node_modules/rxjs";
     import { ITheme } from "~/models/common/theme/ITheme";
     import { IThemeService } from "~/services/theme/IThemeService";
+    import {ICommunity} from "~/models/communities/community/ICommunity";
 
     @Component({
-        name: "OkCategoryButton",
+        name: "OkDynamicStylesButton",
```

Comment:
This name is a bit confusing, weren't you intending to name it something like OkBadge?